contracts/TransmuteFramework/EventStoreFactory.sol

```diff
@@ -47,7 +47,6 @@ contract EventStoreFactory is EventStore {
     EventStoreAddresses.add(address(_newEventStore));
     creatorEventStoreMapping[msg.sender].add(address(_newEventStore));
 
-    uint eventIndex = solidityEventCount;
```

Comment:
the reason was not longer valid once we moves to value events (no need to track the original index)